sr_data_visualization/scripts/sr_data_visualization/sr_data_visualizer_gui.py

```diff
@@ -0,0 +1,1095 @@
+        self.tab_widget_main.setStyleSheet(stylesheet)
+
+        self.tab_widget_main.currentChanged.connect(self.tab_change)
+        self.tabWidget_motor_stats.currentChanged.connect(self.tab_change_mstat)
+
+        motor_stat_keys_file = os.path.join(rospkg.RosPack().get_path('sr_data_visualization'), 'config',
+                                            'data_visualiser_motor_stat_keys.yaml')
+        parameters_file = os.path.join(rospkg.RosPack().get_path('sr_data_visualization'), 'config',
+                                       'data_visualiser_parameters.yaml')
+
+        self.font_offset = -3
+
+        self._widget.resizeEvent = self.on_resize_main
+
+        with open(motor_stat_keys_file, 'r') as stream:
+            try:
+                self.motor_stat_keys = yaml.load(stream)
+            except yaml.YAMLError as exc:
+                print(exc)
+
+        with open(parameters_file, 'r') as stream:
+            try:
+                data_loaded = yaml.load(stream)
+                self.initialize(data_loaded)
+            except yaml.YAMLError as exc:
+                print(exc)
+
+        self.include_tactile_plugin()
+
+        self.setup_radio_buttons()
+
+        self.tabWidget_motor_stats.setCurrentIndex(0)
+        self.tabWidget_motor_stats.setCurrentIndex(1)
+        self.tabWidget_motor_stats.setCurrentIndex(2)
+        self.tabWidget_motor_stats.setCurrentIndex(3)
+        self.tabWidget_motor_stats.setCurrentIndex(4)
+        self.tabWidget_motor_stats.setCurrentIndex(5)
+        self.tabWidget_motor_stats.setCurrentIndex(0)
+
+        self.change_graphs(all=True, type="motor_stat", ncol=1)
+
+        self.t0 = 0
+        self.init_complete = True
+        # TODO: refresh graphs on resize?
+
+    def on_resize_main(self, empty):
+        if (self._widget.width() * self._widget.height()) < 3500000:
+            self.font_offset = -3
+        else:
+            self.font_offset = 1
+
+    def include_tactile_plugin(self):
+        self.tactile_gui_list = []
+        i = 0
+        self.timer = QTimer(self._widget)
+        while i < 5:
+            tactile_gui = SrGuiBiotac(None)
+            tactile_gui._widget = self._widget
+            tactile_gui.find_children(i)
+            self.tactile_gui_list.append(tactile_gui)
+            widget = self._widget.findChild(QWidget, "widget" + "_" + str(i))
+            widget.resizeEvent = self.on_resize_tactile
+
+            self.timer.timeout.connect(widget.update)
+            widget.paintEvent = tactile_gui.paintEvent
+            if not tactile_gui._hand_parameters.mapping:
+                rospy.logerr("No hand detected")
+            i += 1
+
+        self.label_list = []
+        self.lcd_list = []
+
+        j = 0
+        while j < 5:
+            i = 1
+            lcds = []
+            while i < 25:
+                if i < 10:
+                    lcd = self._widget.findChild(QLCDNumber, "lcdE0" + str(i) + "_" + str(j))  # %d_0" + "_" + str(i))
```

Comment:
remove extra comments if they are not needed
```suggestion
                    lcd = self._widget.findChild(QLCDNumber, "lcdE0" + str(i) + "_" + str(j))
```

Reply:
Done